projects/planner/src/app/learn/learn.component.html

```diff
@@ -18,7 +18,8 @@ <h2 class="accordion-header">
                     <li *ngFor="let item of section.items" class="list-group-item px-0">
                       <div
                         mdbRipple
-                        class="d-flex justify-content-between align-items-center px-3 py-2 rounded cursor-pointer"
+                        class="d-flex justify-content-between align-items-center px-3 py-2 rounded "
+                        style="cursor: pointer;"
```

Comment:
dont use styles, if not bottstrap class fits, create class and put it to the css file.